infra/ecs-terraform/service/main.tf

```diff
@@ -0,0 +1,107 @@
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc
```

Comment:
copy paste from the analytics module, which also is "triggered only" (eg. no web server)